### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by corentincarton**

### CODE REVIEW

File: docs/2_requirements/index.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
 # Requirements
 
-
 ## Software required
 
-- pcraster > 0.41 http://pcraster.geo.uu.nl/
-- Python 2.7
-- postscript (need by some postprocessing scripts in order to generate figures, optional) 
+- Python >= 3.8
```
Comment (doc78) [inline docs/2_requirements/index.md:5]: We actually used Python 3.7

Comment (corentincarton) [reply docs/2_requirements/index.md]: Changed.

**Comment by corentincarton** [comment]
@doc78 shall we merge this? I see users are asking questions so maybe this would help?

**Review by doc78 — APPROVED** [review]

**Comment by doc78** [comment]
Merged on develop. Further checks to be done after testing and using the tool for the new calibration before merging in master.

**Pull request merged**